repo_health_dashboard/utils/utils.py

```diff
@@ -51,22 +51,34 @@ def standardize_metadata_by_repo(metadata_by_repo={}):
         output[dict_name] = superset_output
     return output
 
-def squash_and_standardize_metadata_by_repo(metadata_by_repo={}):
+def squash_and_standardize_metadata_by_repo(metadata_by_repo):
     """
     Squashes all metadata_by_repo to only one level and makes sure each has the same keys
     """
     for dict_name, item in metadata_by_repo.items():
         metadata_by_repo[dict_name] = squash_dict(item)
     return standardize_metadata_by_repo(metadata_by_repo)
 
-def write_squashed_metadata_to_csv(metadata_by_repo={}, filename="dashboard.csv"):
+def write_squashed_metadata_to_csv(metadata_by_repo, filename, configuration):
     """
     Assume all the metadata_by_repo have the same keys
     """
-    superset_keys = list(sorted(get_superset_of_keys(metadata_by_repo)))
+    superset_keys = get_superset_of_keys(metadata_by_repo)
+    for key in configuration['check_order']:
+        superset_keys.discard(key)
```

Comment:
A little shorter, if you'd like:

```suggestion
    superset_keys -= configuration['check_order']
```